docs/src/adr04-reactor-stub/adr04-reactor-stub.md

```diff
@@ -0,0 +1,118 @@
+# ADR-04: Atomkraft reactor stub command
+
+## Context
+This ADR describes one component of the command-line tool of Atomkraft.
+The goal of the command-line tool `atomkraft` is to make it easy for users to setup their Atomkraft projects
+and run basic commands.
+
+## Scope
+This is an early version of the ADR, meant for the first prototype.
+Major changes to the scope are very likely.
+
+## Reactor
+
+Once the user has a TLA+ model (either written by hand, or one provided by Atomkraft), they need to write a *reactor*: 
+a set of Python functions connecting the actions of the model to executions of the code.
+
+In this ADR we describe a command `atomkraft reactor ...` which creates a stub of the reactor, 
+auto-generating all the boilerplate, and leaving it to the user to only fill in application-specific parts.
+
+## Task
+
+The interface of the command looks like this: 
+`atomkraft reactor <action-list> <model>.tla [<reactor-stub-file>]`
+where
+ - `action-list` is a list of actions for which to generate stubs
+ - `model` is the TLA model for which we are implementing a reactor
+ - `reactor-stub-file` is a path at which the reactor file should be created. 
+ If omitted, a default path is used: `atomkraft/reactors/<model>_reactor.py`
+
+ ### Config updates
+ This command updates the content of the config file `.atomkraft.toml`.
+ In particular, if updates:
+  - the value of the `model` field
+  - the value of the `reactor` field
+
+The reason for updating the config file is to give users a way to silently refer to the most recent model and reactor files.
+
+
+### Stub content
+The code assumes the access to the `config` dictionary, which has access to everything written in the `.atomkraft` file.
+Furthermore, it assumes imported `Testnet` from `cosmos_net.pytest`.
+
+ The stub should include:
+
+ - a stub for the testnet initialization function, based on the value of the `binary` that was put into `.atomkraft` by the command `atomkraft init`.
```

Comment:
@rnbguy , would like agree with you about the artifacts of the init function. E.g., will it save only the binary name, or all the values necessary to span the default testnet.

If former, then we should figure out where to keep default values for familiar binaries.